lib/internal/main/print_help.js

```diff
@@ -164,7 +164,7 @@ function print(stream) {
   options.set('--', { helpText: 'indicate the end of node options' });
   stream.write(
     'Usage: node [options] [ -e script | script.js | - ] [arguments]\n' +
-    '       node inspect script.js [arguments]\n\n' +
+    '       node inspect [script.js | -e "script" | <host>:<port>] …\n\n' +
```

Comment:
This uses an ellipsis character (_…_) where everywhere else, we use three dots (_..._). On the one hand, the ellipsis is more correct. On the other hand, is there a situation where it will be an unprintable character because the terminal only accepts ASCII? Regardless, the code should probably pick one or the other and stick with it.

Reply:
Linter requires non-ASCII characters (to keep binary size down, believe it or not) so `...` it must be.

Also, this should probably align with line 166 in that no `"` around `script` and `-e` comes before the `script.js` option.

Assuming you left the default checked state for the checkbox about allowing collaborators in this repo to push to your branch, I'll make these updates right now....